src/components/atoms/switch/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import clsx from "clsx"
+import * as _Switch from "@radix-ui/react-switch"
+
+/**
+ * A controlled switch component atom.
+ */
+function Switch(props: _Switch.SwitchProps) {
+  const { checked } = props
+
+  return (
+    <_Switch.Root
+      {...props}
+      className={clsx("w-8 h-[18px] rounded-full", {
```

Comment:
We could also add a transition to the background color as well

We could consider adding this to the `tailwind.config.ts` as well in the `transitionProperty`s. I'm thinking we could add a `transition-bg` class for background color animation: 
```javascript
 transitionProperty: {
        ...
        bg: "background-color",
      },
```
 wdyt?